content/zh-cn/examples/pods/pod-with-scheduling-gates.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
This is supposed to be localized with `content/en/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md`.